### PR TITLE
[10.0][FIX]pylint of many modules

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
I want this green

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
somuchwin

**Review by MiquelRForgeFlow — APPROVED** [review]
why don't merged this? 😂

File: .travis.yml [hunk 1]
```diff
@@ -5,10 +5,13 @@ cache:
   directories:
     - $HOME/.cache/pip
 
+postgres: 9.6
```
Comment (MiquelRForgeFlow) [inline .travis.yml]: don't needed here if it's already put below

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (AaronHForgeFlow) [reply .travis.yml]: oh I see. thanks for the info

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
:green_heart:

**Comment by MiquelRForgeFlow** [comment]
still_wainting_merge

**Pull request closed without merging**